### PR TITLE
Update/tests

### PR DESCRIPTION
**Pull request by circlecube**
## Proposed changes

This updates the tests that have been failing. It should speed them up as well. 

I've implemented the `testIsolation` flag so each test is expected to be more independent, and combined tests that were not independent to run in the same `it`. (We are working through all tests to use this recommended setting so I went ahead and added while I was working on these.)

I combined some cli commands too so they should run faster. These were eating up a lot of time running on and before each test step and many were repeated so I was able to combine test steps and cli commands.

I also started adding some data testid attributes to elements which can be used once they are included in the build, I didn't switch to those yet though they are commented out for selecting the nowoo-store-title and description, they should be a faster and clearer way to select elements on the page.

This also includes some lint fixes my IDE changed, I hope it doesn't make review confusing.

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [ ] Linting and tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by circlecube** [comment]
These are now finally passing tests. I converted all but one of the tests to use `testIsolation` - and the remaining one (ecommerce-next-steps) I didn't update it because it does not pass locally for me, I assume I need to start with a fresh install so the next steps display properly.

Friendly reminder that new features should have new tests and PRs shouldn't be merged unless tests pass. We certainly should not release a module with failing tests because that cascades to every other module and plugin displaying failing tests.

**Review by wpalani — DISMISSED** [review]

File: tests/cypress/integration/Site-Capabilities/siteCapabilities.cy.js [hunk 1]
```diff
@@ -1,149 +1,119 @@
...
+			wpLogin();
+			uninstallPlugins();
+			installWoo();
+			cy.visit( '/wp-admin/admin.php?page=' + pluginId );
+		} );
 
-	beforeEach( () => {
-		cy.visit( '/wp-admin/admin.php?page=' + pluginId + '#/home' );
-	} );
+		before( () => {
```
Comment (sangeetha-nayak) [inline tests/cypress/integration/Site-Capabilities/siteCapabilities.cy.js:44]: HG run has a syntax error, please fix it @circlecube

File: tests/cypress/integration/wp-module-support/utils.cy.js [hunk 2]
```diff
@@ -71,3 +77,15 @@ export const waitForNextSteps = () => {
...
+	cy.log( 'Deleting capabilities transient' );
+	cy.exec(
+		`npx wp-env run cli wp transient delete nfd_site_capabilities`,
+		{ failOnNonZeroExit: false }
+	);
+};
+
+export const wpLogin = () => {
+	cy.login( Cypress.env( 'wpUsername' ), Cypress.env( 'wpPassword' ) );
+};
```
Comment (sangeetha-nayak) [inline tests/cypress/integration/wp-module-support/utils.cy.js:91]: add a line break.

**Review by sangeetha-nayak — CHANGES_REQUESTED** [review]

File: tests/cypress/integration/wp-module-support/utils.cy.js (hunk 2 above)
Comment (circlecube) [reply tests/cypress/integration/wp-module-support/utils.cy.js]: Again, thanks for the callout. This is updated.

**Review by sangeetha-nayak — APPROVED** [review]

**Review by sangeetha-nayak — APPROVED** [review]

**Pull request merged**